### PR TITLE
6 add docker configuration

### PR DESCRIPTION
**Pull request by lukaszmindpal**

### CODE REVIEW

File: dev.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node:carbon as builder
+
+WORKDIR /usr/src/app/
```
Comment (grzegorz-bielski) [inline dev.Dockerfile:3]: why such a dir? 👀

File: prod.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM node:carbon as builder
+
+WORKDIR /usr/src/app/
+
+COPY ./package.json ./
+
+RUN npm install
```
Comment (grzegorz-bielski) [inline prod.Dockerfile:7]: `npm ci`

File: prod.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+FROM node:carbon as builder
+
+WORKDIR /usr/src/app/
+
+COPY ./package.json ./
```
Comment (grzegorz-bielski) [inline prod.Dockerfile:5]: `COPY package*.json ./`

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      - frontend
+
+  nginx:
+    container_name: mask-app-nginx
+    build:
+      context: ./nginx
+    ports:
+      - 8080:80
+      - '8081:8081'
+    restart: always
```
Comment (grzegorz-bielski) [inline docker-compose.yml:16]: `restart: unless-stopped`

**Review by grzegorz-bielski — APPROVED** [review]
LGTM, just a few things to consider

File: dev.Dockerfile (hunk 1 above)
Comment (lukaszmindpal) [reply dev.Dockerfile:3]: I prefer /usr/src/ because this is the default directory for deploy application etc

**Comment by grzegorz-bielski** [comment]
:shipit:

**Pull request merged**